### ISSUE TITLE
Add rebuilding documentation to fabric script.

### ISSUE THREAD

**Issue by kmonson** [Electrostatics/apbs-pdb2pqr#355]
A little harder than it looks at first glance. Some of the scripts for that purpose will have to be rewritten to fit in the scons way of doing things.

**Comment by keith923** [comment]
How has it been done up until now?  My first reaction is that we put this off if it's currently working.  Making things work with scons seems anti-productive to me in the light of eventually making pdb2pqr a plugin in a totally different architecture.

**Comment by kmonson** [comment]
It's a separate script that must be run by hand. In PDB2PQR 2.0 I forgot to run it and so the docs might be out of date in the released version. It's all API stuff so it's not a huge deal, but it is irksome that I forgot it. It is a step in the release SOP and I just glazed right over it. 

I could also put it in the fabric script that I use to do releases. In fact I'll change the issue accordingly.

**Comment by lizutah** [comment]
@kmonson - I think you fixed this issue with your last update to the fab script?

**Comment by kmonson** [comment]
No, it's not in there yet. I'm not sure if it's a good idea to put it in.
The results need to be checked in after it's run. When doing a release the
fab script is really meant to be run on a clean checkout. This ensures that
what we are tagging is exactly what is going into the source zip file.

I ran into some problems when I built the docs last. The SOP calls for
running it after doing a clean and this no longer works. Fortunately
running on the code base after a build does not seem to be a big deal. (It
I seem to remember that it used to create lots superfluous pages, I don't
know what changed.)
